util/git/git.go

```diff
@@ -11,20 +12,59 @@ import (
 
 // NormalizeGitURL normalizes a git URL for lookup and storage
 func NormalizeGitURL(repo string) string {
```

Comment:
Wondering about unit tests for this method, too.  We can always add later.

Reply:
Done.